dts/arm/nxp/nxp_ke15z256vlx7.dtsi

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <nxp/nxp_ke1xz.dtsi>
+
+/ {
+	sram_l: memory@1fffe000 {
+		compatible = "zephyr,memory-region", "mmio-sram";
+		reg = <0x1FFFE000 DT_SIZE_K(8)>;
+		zephyr,memory-region = "SRAML";
+	};
+
+	sram0: memory@20000000 {
```

Comment:
should call this sram_u to be consistent

Reply:
Stay consistent in what context as this is how the nodes are laid out in the nxp_k6x.dtsi?

Reply:
it is called sram_u in the RM